src/globals.d.ts

```diff
@@ -0,0 +1,5 @@
+declare global {
+  interface Window extends IWindowGlobals {
+    SPOTIM: any;
```

Comment:
Can you type the functions we are using on this package?

Reply:
sure